### PR TITLE
Added Line Rounding Option

### PR DESCRIPTION
**Pull request by Allstreamer**
Users can now chose if they prefere Flat Line endings or rounded ones

- Localization:
English
German

### CODE REVIEW

**Comment by mbrlabs** [comment]
I think it makes more sense to put this option into the "Rendering" tab below the Anti-Aliasing method. The general settings for brushes just define default values for the tools on startup, not how the stroke is rendered...so it doesn't really fit in there.

**Comment by Allstreamer** [comment]
Okay i'll move it over.

**Comment by Allstreamer** [comment]
There we go

File: lorien/Misc/Settings.gd [hunk 1]
```diff
@@ -5,6 +5,7 @@ const GENERAL_DEFAULT_CANVAS_COLOR 		:= "general_default_canvas_color"
 const GENERAL_DEFAULT_BRUSH_SIZE 		:= "general_default_brush_size"
 const GENERAL_DEFAULT_BRUSH_COLOR 		:= "general_default_brush_color"
 const GENERAL_DEFAULT_PROJECT_DIR		:= "general_default_project_dir"
+const GENERAL_BRUSH_ROUNDING			:= "general_brush_rounding"
```
Comment (mbrlabs) [inline lorien/Misc/Settings.gd:8]: This should be `RENDERING_BRUSH_ROUNDING` and `rendering_brush_rounding` now

**Review by mbrlabs — CHANGES_REQUESTED** [review]

**Pull request merged**